dias/analyzers/sensitivity_analyzer.py

```diff
@@ -40,7 +40,13 @@ class SensitivityAnalyzer(CHIMEAnalyzer):
 
     Metrics
     ----------
-    None
+    dias_task_<task name>_average_sensitivity
```

Comment:
```suggestion
    dias_data_<task name>_average_sensitivity
```

Since you use `add_data_metric`, the name will be `dias_data_*`.
see https://dias.readthedocs.io/en/latest/dev/analyzer.html#other-useful-utilities-dias-provides